src/wymeditor/plugins/table/jquery.wymeditor.table.js

```diff
@@ -138,22 +138,38 @@ TableEditor.prototype.bindEvents = function () {
     jQuery(wym._box).find(
         tableEditor._options.sAddRowButtonSelector
     ).click(function () {
-        return tableEditor.addRow(wym.selectedContainer());
+        var cont = wym.selectedContainer();
+        if (cont) {
+            tableEditor.addRow(cont);
+        }
```

Comment:
I've also added a conditional, here. It is important, for when the call to `selectedContainer` during some selections returns `false`.

That currently produces an error inside the add/remove row/column methods.

This fixes it.